### ISSUE TITLE
Color Corrector doing absolutely nothing

### ISSUE THREAD

**Issue by ultramizer** [regiellis/ComfyUI-EasyColorCorrector#13]
I have tried auto mode and also many presets. I'm working with regular, well lit, business portraits. No matter what settings i use, if ai analysis on or off, effect_strenght or enhancement_strenght at any values, nothing happens. I use an image comparison node and it's exactly the same. In the log i can see that the AI Detection detects it as a portrait and it also extracts some color HEX codes, and i can see the histogram if i enable it. So there's no error and it runs smoothly but just nothing happens.

**To Reproduce**
1. Put node into workflow
2. Connect image in and out
3. Nothing happens to the image

**Expected behavior**
Something happens to the image!

**Screenshots**
I can't really show you anything since nothing happens to the image.

**Desktop (please complete the following information):**
Latest everything, latest comfyui, latest node, latest Windows, latest Chrome

**Comment by firsack** [comment]
same thing in my case.
tried Batch Color Corrector (beta). no matter what parameters I use - no effect on anything at all. 

ComfyUI version: 0.14.1
Python version: 3.13.11
pytorch version: 2.10.0+cu130
RTX 4090

using a preset:
```
⚠️ Minimal/no changes detected
🔄 GPU Processing batch 33/33: frames 512-520
🔥 Processing batch of 9 frames on cuda:0
🎯 Batch tensor device: cuda:0
🎛️ Parameters: warmth=0.0, vibrancy=0.0, brightness=0.0, contrast=0.0
🎨 Applied batch preset: Vibrant Concept -> Vibrant Concept
📊 Original batch mean: 0.4308
📊 Final batch mean: 0.4308 | Change: 0.000000
```


manual:
```
got prompt
🔧 CUDA Available: True
🔧 User GPU preference: False
🔧 Initial device: cpu
🔧 Input tensor device: cpu | dtype: torch.float32
💻 User selected CPU processing - keeping tensors on CPU
🎬 GPU Batch Color Corrector: Processing 521 frames (928x736) on cpu
📊 Processing in batches of 16 frames
🔄 GPU Processing batch 1/33: frames 0-15
🔥 Processing batch of 16 frames on cpu
🎯 Batch tensor device: cpu
🎛️ Parameters: warmth=0.0, vibrancy=0.0, brightness=1.0, contrast=0.0
📊 Original batch mean: 0.3790
❌ Error during batch processing: name 'tint' is not defined
Prompt executed in 7.79 seconds
```

**Comment by timoschaffner** [comment]
I gave up on this since it seems this Node is dead. I switched over to a workflow with Flux2 Image Editing or Flux2 Context respectively. Gives much better results but sometimes also changes too much. But at least i get some outcome :)
Just thought i might leave this here for others looking to change something related to color or style within their pictures.

**Comment by neurocontrarian** [comment]
I can confirm the exact same issue. I'm using EasyColorCorrector after a FLUX.2 Klein 9B pipeline for vintage photo restoration. No matter what I change — Auto mode, Preset mode, AI analysis on/off, effect_strength, white_balance_strength — the output image is virtually identical to the input.

Interestingly, in Preset mode, the image does change slightly, but all presets produce the same result regardless of which one is selected (Film Noir, Sepia, etc. all look identical).

The logs show everything is running correctly — AI analysis detects the scene, faces are found, GPU is being used — but the node just doesn't apply any visible correction.

The only way I've found to get any change in the output is to modify the noise seed in a preceding node by 1 unit between each run. It seems like the node is aggressively caching its result and ignoring parameter changes entirely.

Environment: Ubuntu, ComfyUI latest, RTX 3090, latest version of the node (commit fe86b8a on main branch). All dependencies installed (timm, scikit-learn, scikit-image, colour-science, OpenEXR, rawpy, OpenCV).

**Comment by j2l** [comment]
check this fork: https://github.com/AlwaysSolo/ComfyUI-ColorCo
But batch is not working for me.

**Comment by LeeC2202Git** [comment]
I thought I was using it wrong until I used the Black & White preset and it came out in colour.

**Comment by regiellis** [comment]
Hey Guys,
The node is not dead...just been slow to update for several reasons...the biggest of them being that the node needs to be moved to v3 of the API and the change in how models handle color within models, which means a complete re-write. I have been slowly working on this...but it is not a priority given the current ComfyUI eco-system, open-source and SWE in general.  

So is it dead, no...I may push some alpha v2 code...but my focus is putting food on the table for the fam and open source just doesn't do that these days and the ComfyUI guys are not in a hurry to support node creators.

**Comment by Dark-Lord-9** [comment]
Batch images doesn't work for me. Just like the others, batch image node appears to do nothing.
Trying to load a multi-image batch into the original node causes any image after the first to bleed into each other.
Single images work fine and i'm pretty happy with the results. Just can't use a batch count higher than 1, which is frustrating.
